src/pygeoprocessing/geoprocessing.py

```diff
@@ -2631,17 +2643,13 @@ def convolve_2d(
                     os.path.basename(target_path)),
                 _LOGGING_PERIOD)
         # delete the mask raster
```

Comment:
Should this comment make more sense now as: dereference mask_raster

Reply:
It actually is deleted though, see `os.remove(mask_raster_path)`. I suppose that doesn't need a comment though? Removed in 05572f475c42081e9242b4b01e534cf7cc067399